docs/source/pillars/supporting.md

```diff
@@ -0,0 +1,217 @@
+  - This includes ensuring all documentation is discoverable, clear, and able to be easily updated based on stakeholder feedback
+  - Mandatory
+* - You should have an education programme in place to upskill stakeholders in the use and management of your TRE infrastructure
+  - This may include learning modules, workshops and other resources on how to effectively access and use a TRE, FAQs pages, and accessible pathways for additional support
+  - Recommended
+* - You should periodically carry out a training needs analysis (TNA) for all stakeholders included within your TRE provision
+  - At least once every 12 months you should assess the training needs of your stakeholders, and ensure they have easy access to all required training materials
+  - Recommended
+```
+
+## Financial management
+
+_All activities aimed at the efficient and effective management of money (funds) in such a manner as to allow the organisation to accomplish its objectives._
+
+```{list-table}
+:header-rows: 1
+:name: tab-financial-management
+
+* - Statement
+  - Guidance
+  - Importance
+* - All projects requiring a TRE must know and accept the financial cost of doing so
+  - Costs may include provision of the underlying TRE infrastructure, additional resources required in a specific TRE (for instance memory or additional compute), hardware including managed devices, and staff support costs
+  - Mandatory
+* - You should be able to track costs associated with each TRE project
+  - This includes knowing which costs are associated with which project, and having an appropriate charging mechanism in place in line with your organisational policy
+  - Recommended
+* - You should have a process in place to ensure your TRE provision remains sustainable
+  - This could include having a cost recovery process in place, or setting up a long-term funding mechanism to support projects with TREs. At any given time, you should have funds free to cover all potential foreseen TRE provision for at least 12 months
+  - Recommended
+* - You should minimise the cost of your TRE infrastructure wherever possible
+  - You should have regular reviews of your TRE provision and actively work to bring down costs, streamline provision, and optimise support
+  - Optional
+```
+
+## Procurement
+
+_What the organisation does to ensure the effective sourcing, purchasing and supply of the goods and services that enable the organisation to operate._
+
+```{list-table}
+:header-rows: 1
+:name: tab-procurement
+
+* - Statement
+  - Guidance
+  - Importance
+* -
+  -
+  -
+```
+
+## IT Service management
+
+_The implementation and management of quality IT services that meet the needs of the organisation._
+
+```{list-table}
+:header-rows: 1
+:name: tab-it-service-management
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must have a team in place to support projects working with TREs
```

Comment:
Is this separate from the "project management" support listed earlier?

Reply:
I see it as yes, one is ensuring the project is running smoothly, the other is to ensure there is technical support in place